includes/sanitizers/class-amp-block-sanitizer.php

```diff
@@ -76,10 +76,11 @@ static function ( $class ) use ( &$responsive_width, &$responsive_height ) {
 			// @todo Should we consider just eliminating the .wp-block-embed__wrapper element since unnecessary?
 			// For visual parity with blocks in non-AMP pages, override the oEmbed's natural responsive dimensions with the aspect ratio specified in the wp-embed-aspect-* class name.
 			if ( $responsive_width && $responsive_height ) {
-				$amp_element = $this->dom->xpath->query( './div[ contains( @class, "wp-block-embed__wrapper" ) ]/*[ @layout = "responsive" ]', $node )->item( 0 );
+				$amp_element = $this->dom->xpath->query( './div[ contains( @class, "wp-block-embed__wrapper" ) ]/*[ @layout = "responsive" or @layout = "intrinsic"  ]', $node )->item( 0 );
```

Comment:
Alternatively you could use:

```suggestion
				$amp_element = $this->dom->xpath->query( './div[ contains( @class, "wp-block-embed__wrapper" ) ]/*[ @layout = ( "responsive" or "intrinsic" )  ]', $node )->item( 0 );
```

Reply:
Interesting. I wasn't aware that works in XPath. So it's distinctly not like PHP where this definitely wouldn't work:

```php
if ( $layout === ( 'responsive' || 'intrinsic' ) ) {
```

Where instead you'd need to do:

```php
if ( in_array( $layout, [ 'responsive', 'intrinsic' ], true ) ) {
```

That seems like somewhat of an esoteric feature of XPath. Where did you come across it?

Reply:
Yea since XPath is it's own query language I doubt PHP would have any influence on the query itself. I have a cheatsheet that I refer to from time to time when I'm writing queries: https://devhints.io/xpath.

Reply:
Some examples there are only XPath 2.0 compatible so YMMV.

Reply:
I actually don't see the `@layout = ( "responsive" or "intrinsic" )` syntax mentioned on that cheatsheet.

Reply:
In any case, I like the conciseness, but given the boolean logic is foreign to the syntax of how other languages do booleans, I think I'll leave it as is.

Reply:
> I actually don't see the @layout = ( "responsive" or "intrinsic" ) syntax mentioned on that cheatsheet.

Oh right, that was inspired by some of the logic operator examples on the page. The syntax is indeed a bit esoteric so I'm fine with what's there currently.